src/main/java/com/prgrms/kream/domain/style/facade/StyleFacade.java

```diff
@@ -36,105 +36,99 @@ public class StyleFacade {
 	private final ImageService imageService;
 
 	@Transactional
-	public RegisterFeedFacadeResponse register(RegisterFeedFacadeRequest registerFeedFacadeRequest) {
-		RegisterFeedFacadeResponse registerFeedFacadeResponse = toRegisterFeedFacadeResponse(
-				styleService.register(
-						toRegisterFeedServiceRequest(registerFeedFacadeRequest)
+	public FeedRegisterFacadeResponse registerFeed(FeedRegisterFacadeRequest feedRegisterFacadeRequest) {
+		FeedRegisterFacadeResponse feedRegisterFacadeResponse = toFeedRegisterFacadeResponse(
+				styleService.registerFeed(
+						toFeedRegisterServiceRequest(feedRegisterFacadeRequest)
 				)
 		);
 
 		// 이미지 등록 서비스 호출
 		imageService.registerImage(
-				registerFeedFacadeRequest.images(),
-				registerFeedFacadeResponse.id(),
+				feedRegisterFacadeRequest.images(),
+				feedRegisterFacadeResponse.id(),
 				DomainType.FEED
 		);
 
-		return registerFeedFacadeResponse;
+		return feedRegisterFacadeResponse;
 	}
 
 	@Transactional(readOnly = true)
-	public GetFeedFacadeResponses getTrendingFeeds(GetFeedFacadeRequest getFeedFacadeRequest) {
-		return merge(styleService.getTrendingFeeds(toGetFeedServiceRequest(getFeedFacadeRequest)));
+	public FeedGetFacadeResponses getAllFeeds(FeedGetFacadeRequest feedGetFacadeRequest) {
+		return merge(styleService.getAllFeeds(toFeedGetServiceRequest(feedGetFacadeRequest)));
 	}
 
 	@Transactional(readOnly = true)
-	public GetFeedFacadeResponses getNewestFeeds(GetFeedFacadeRequest getFeedFacadeRequest) {
-		return merge(styleService.getNewestFeeds(toGetFeedServiceRequest(getFeedFacadeRequest)));
+	public FeedGetFacadeResponses getAllFeedsByTag(FeedGetFacadeRequest feedGetFacadeRequest, String tag) {
+		return merge(styleService.getAllFeedsByTag(toFeedGetServiceRequest(feedGetFacadeRequest), tag));
 	}
 
 	@Transactional(readOnly = true)
-	public GetFeedFacadeResponses getAllByTag(GetFeedFacadeRequest getFeedFacadeRequest, String tag) {
-		return merge(styleService.getAllByTag(toGetFeedServiceRequest(getFeedFacadeRequest), tag));
+	public FeedGetFacadeResponses getAllFeedsByMember(FeedGetFacadeRequest feedGetFacadeRequest, Long memberId) {
```

Comment:
```suggestion
	public FeedGetFacadeResponses getAllFeedsByMemberId(FeedGetFacadeRequest feedGetFacadeRequest, Long memberId) {
```
memberId라고 하면 더 어색할까요?

Reply:
그게 좀 더 명확할 것 같네요! 반영하겠습니다